src/Components/AdvancedSearch/Internal/QueryBuilder/QueryBuilder.types.ts

```diff
@@ -96,9 +118,20 @@ export interface PropertyOption {
     data: {
         name: string;
         type: PropertyValueType;
+        icon: string;
```

Comment:
nit: `iconName` would be more clear since I'm sure you pass this into the Icon control.